server/amp/handlers/story-page.js

```
@@ -144,6 +144,10 @@ async function ampStoryPageHandler(
       });
       merge(mergedAdditionalConfig, additionalConfig, fetchedAdditionalConfig);
     }
+    // the query appending happens in the worker, this is needed for any publisher who needs ad-free in amp story
+    if(req && req.query && req.query.subscriber === "true") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)